images/win/scripts/Tests/Tools.Tests.ps1

```diff
@@ -14,4 +14,90 @@ Describe "Kind" {
     It "Kind" {
         "kind version" | Should -ReturnZeroExitCode
     }
+}
+
+Describe "Jq" {
+    It "Jq" {
+        "jq -n ." | Should -ReturnZeroExitCode
+    }
+}
+
+Describe "Julia" {
+    It "Julia path exists" {
+        "C:\Julia" | Should -Exist
+    }
+
+    It "Julia" {
+        "julia --version" | Should -ReturnZeroExitCode
+    }
+}
+
+Describe "Mercurial" {
+    It "Mercurial" {
+        "hg --version" | Should -ReturnZeroExitCode
+    }
+}
+
+Describe "KubernetesCli" {
+    It "kubectl" {
+        "kubectl version --client=true --short=true" | Should -ReturnZeroExitCode
+    }
+
+    It "minikube" {
+        "minikube version --short" | Should -ReturnZeroExitCode
+    }
+}
+
+Describe "Mingw64" {
+    It "gcc" {
+        "gcc --version" | Should -ReturnZeroExitCode
+    }
+
+    It "g++" {
+        "g++ --version" | Should -ReturnZeroExitCode
+    }
+
+    It "make" {
+        "make --version" | Should -ReturnZeroExitCode
+    }
+}
+
+Describe "InnoSetup" {
+    It "InnoSetup" {
+        (Get-Command -Name iscc).CommandType | Should -BeExactly "Application"
+    }
+}
+
+Describe "GitHub-CLI" {
+    It "gh is located in ${env:ProgramFiles(x86)}\GitHub CLI" {
+        "${env:ProgramFiles(x86)}\GitHub CLI\gh.exe" | Should -Exist
```

Comment:
Just for curiosity: why it is important to check the directory?

Reply:
Fixed.